session/bootstrap.go

```diff
@@ -209,6 +209,20 @@ const (
 		feedback blob NOT NULL,
 		index hist(table_id, is_index, hist_id)
 	);`
+
+	// CreateBindInfoTable stores the sql bind info which is used to update globalBindCache.
+	CreateBindInfoTable = `CREATE TABLE IF NOT EXISTS mysql.bind_info (
+		original_sql varchar(2048) NOT NULL  ,
+      	bind_sql varchar(2048) NOT NULL ,
+      	default_db varchar(1024)  NOT NULL,
+		status int(10) NOT NULL DEFAULT 1,
+		create_time timestamp NOT NULL,
+		update_time timestamp NOT NULL,
+		charset varchar(100) NOT NULL,
+		collation varchar(100) NOT NULL,
+ 		UNIQUE KEY sql_db (original_sql, default_db),
+		INDEX time_index(update_time)
+	) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;`
```

Comment:
how about defining a sharding row bits to avoid hot write?

Reply:
The sql bind is only used by tidb administrator, so I think a table is enough.

Reply:
We may need to use one TiDB cluster as the backup cluster, and import data from another TiDB cluster, if there are a lot of sql bindings, there might be a hot write problem?

Reply:
I don't think a tidb will have so much sql bind info